src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/payments/client/FeeClientTest.java

```diff
@@ -86,4 +103,8 @@ private String contestedUri() {
             : serviceConfig.getContestedKeyword());
     }
 
+    private String contestedSchedule1Uri() {
+        return "http://localhost:8182/fees-register/fees/lookup?service=private%20law&jurisdiction1=family&jurisdiction2=family-court&channel=default"
```

Comment:
Hard coded URL could this be placed in application.propertes file for test resources?